standards/java_coding_standards.md

```diff
@@ -0,0 +1,265 @@
+﻿# Java Coding Standards
+
+These standards are a slightly updated version of the standards used in practice for many years in the Environment Agency during the period when Java was defined as the strategic programming language. As such they are comprehensive, mature and follow standard industry practice.
+
+For users of [Eclipse](https://en.wikipedia.org/wiki/Eclipse_(software)) we have [guidance on how to auto-format your code](/guides/plsql_auto_format_toad.md) that applies the standards described below.
```

Comment:
I've fixed the first (👍 for spotting it).

On your second point, is that something specific to the Eclipse IDE / formatter? In which case I'd suggest not worrying about it here but putting something into PR #29 . But I'm not the Java expert, nor the originator of this PR. So this is probably one for @nigejohnson or @sgardnerdell-ea to make the final decision on

Personally (climbs on a soapbox) I'm already a little worried about peppering standards with references to specific tools (for example its been a while since I used or saw Eclipse being used in the wild) hence my inclination to avoid putting too much here.

Reply:
It looks like @<!-- -->formatter tags can be used to disable code formatting rules for specific blocks of code in Eclipse and IntelliJ.  I haven't found anything suggesting it is applicable in Visual Studio Code to date.

I agree that IDE specific guidance should be located in IDE specific documentation. As such maybe this document should have general guidance about disabling formatting in specific cases (such as when using Reactive Java) with signposting for IDE specific instructions.